client/components/books-list.jsx

```diff
@@ -1,28 +1,39 @@
 import React from 'react';
 
 function Book(props) {
-  return (
-    <li className='flex margin-top'>
-      <img src={props.book.book_image} alt={props.book.title}/>
-      <div className='content-holder'>
-        <p className='number-of-weeks'>{props.book.weeks_on_list} WEEKS ON THE LIST</p>
-        <h3 className='title margin-top '>{props.book.title}</h3>
-        <p className='author margin-top '>by {props.book.author}</p>
-        <p className='description margin-top '>{props.book.description}</p>
-      </div>
+  const imgageUrl = props.book.book_image;
+  const numberWeeks = props.book.weeks_on_list;
+  if (props.book.isbns[0]) {
+    const isbn = props.book.isbns[0].isbn10;
+    return (
+      <a
+        href={`#details?isbn=${isbn}&imageurl=${imgageUrl}&n=${numberWeeks}`}
+      className='flex margin-top a-book'>
+        <li className='flex margin-top'>
+          <img src={imgageUrl} alt={props.book.title} />
+          <div className='content-holder'>
+            <p className='number-of-weeks '>{props.book.weeks_on_list} WEEKS ON THE LIST</p>
+            <p className='title margin-top'>{props.book.title}</p>
+            <p className='author margin-top'>by {props.book.author}</p>
+            <p className='description margin-top'>{props.book.description}</p>
+            <p href={`#details?isbn=${isbn}`} className='more-details-p'>More Details...</p>
+          </div>
+        </li>
+      </a>
 
-    </li>
-  );
+    );
+  } else {
+    delete props.book.isbns[0];
+  }
```

Comment:
If we can guarantee that `props.book` will have the correct isbn, then you can remove this `if`.

Reply:
<img width="800" alt="Screen Shot 2022-09-15 at 9 29 09 AM" src="https://user-images.githubusercontent.com/106217355/190457973-47714003-d906-475c-a7df-fca7d6361521.png">